pkg/auth/ldaputil/query.go

```diff
@@ -104,11 +105,17 @@ type LDAPQueryOnAttribute struct {
 // NewSearchRequest creates a new search request from the identifying query by internalizing the value of
 // the attribute to be filtered as well as any attributes that need to be recovered
 func (o *LDAPQueryOnAttribute) NewSearchRequest(attributeValue string, attributes []string) (*ldap.SearchRequest, error) {
+	removeSpaces := func(r rune) rune {
+		if unicode.IsSpace(r) {
+			return -1
+		}
+		return r
+	}
 	if strings.EqualFold(o.QueryAttribute, "dn") {
 		if _, err := ldap.ParseDN(attributeValue); err != nil {
```

Comment:
Can we use result of this function instead of manually parsing and checking validity?